std_msgs/Header header
bool validity
float64 x
float64 y